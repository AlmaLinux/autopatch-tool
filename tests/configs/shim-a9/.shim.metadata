9ca9cfa834aedfaf3efe2216bfa1cb7c286ee1c0 SOURCES/fbx64.efi
5eb0ac78eee6aeeaf44a3f11d002b4fe00af6916 SOURCES/mmx64.efi
4312f246b6ba692040383f10358ac9a5927207de SOURCES/shimaa64.efi
783fb77783e9d0c4c400b723dfd0f02f006616ae SOURCES/shimx64.efi
