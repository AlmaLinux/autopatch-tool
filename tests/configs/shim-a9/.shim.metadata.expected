4591a3da03f337b27e963b69202f7109496c0ef8 SOURCES/fbaa64.efi
403a4e918a00f5e90551085c7ba545969feee26e SOURCES/fbx64.efi
48fe2246bcc52729315e7db8361d753ab6be1633 SOURCES/mmaa64.efi
08fa765dbe8902c022fe0d5b5803c14bc1e44a5c SOURCES/mmx64.efi
206b13df7ccae83bf2f961da7010e16eac372b28 SOURCES/shimaa64.efi
1208043ca28f8566c115d8a15ba9bd8163b6badb SOURCES/shimx64.efi
